java/test/jmri/jmrit/display/layoutEditor/LayoutEditorToolsTest.java

```diff
@@ -21,17 +21,25 @@ public void testCtor() {
         LayoutEditor e = new LayoutEditor();
         LayoutEditorTools t = new LayoutEditorTools(e);
         Assert.assertNotNull("exists", t);
-        JUnitUtil.dispose(e);
+        e.dispose();
     }
 
     // from here down is testing infrastructure
     @Before
     public void setUp() throws Exception {
-        JUnitUtil.setUp();
+        apps.tests.Log4JFixture.setUp();
+        // dispose of the single PanelMenu instance
+        jmri.jmrit.display.PanelMenu.dispose();
```

Comment:
Does this line work in headless mode?

Also, revert to JUnitUtil.setUp() here.

Reply:
PanelMenu.dispose() works in headless mode; however, it should just be removed, since PanelMenu.dispose is a deprecated empty method that does nothing.

Reply:
Done!